lib/client.js

```diff
@@ -118,7 +118,7 @@ class Metacom extends EventEmitter {
           if (stream) {
             console.error(new Error(`Stream ${name} is already initialized`));
           } else {
-            const stream = new MetaReadable({ id, name, size });
+            const stream = new MetaReadable(id, name, size);
```

Comment:
See MetaReadable constructor signature: https://github.com/metarhia/metacom/blob/86504033966fa73b3cb6733456c8d5ba6ced45af/lib/streams.js#L28-L29

Reply:
In client as well

Reply:
@tshemsedinov what exactly you want me to see? we had invalid instantiation and I've fixed it

Reply:
@tshemsedinov MetaReadable accepts id, name and size as positional args. Before my update it was initialized with object as a first arg.

Reply:
Parameters in MetaReadable constructor are separate, not grouped into object, or not "named"

Reply:
Oh, sorry, my fault, yes, landing both commits